app/assets/stylesheets/application.scss

```diff
@@ -113,17 +113,25 @@ h5 {
 }
 
 .div-bar {
-  background-color: #37becc;
+  background-color: $primary-color;
   color: #ffffff;
   padding: 10px;
   margin: 0px;
   margin-top: 15px;
+  box-shadow: 5px 5px 5px #b1b1b1;
```

Comment:
Color literals like `#b1b1b1` should only be used in variable declarations; they should be referred to via variable everywhere else.